### DISCUSSION TITLE
Adding a changelog feature. Worth it or keep it simple?

### DISCUSSION THREAD

**Discussion by HiFiiDev** [thelearn-tech/Apache-configure#6] [Ideas]
So in my repos, I have a changelog script that generates a changelog and also have some various customizability. https://github.com/HiFiiDev/ROM-Changes

Would you like me to add a slimmed down version of a changelogger in your script? Like when the script first gets ran, add a prompt and ask the user if they'd like to see a changelog. If yes, run it (but keep it simple/quick), if not, move on and ask

Perhaps the second time running the script, ask one more time, and if yes, ask if they'd like a changelog generated every launch.

Let me know it shouldn't be too hard

Either way, if you'd ever want to have something like that in your scripts, let me know 😎

**Comment by thelearn-tech** [comment]
NO.
BECAUSE I think it will be advertisement in installation script which  people really wouldn't want  .

But I know how much work is creating a script
So I will put a link to you ROM_change repository

**Reply by HiFiiDev** [reply]
** no self promotion of any kind. I am not that kind of person and am trying to contribute, not promote myself.  Apologise if my request came off that way though.. **

Heres what I said in the issue section too:

It would not be to promote my script.  I would just add a block code in your script to add a changelog feature :)

I will probably just be using [this](https://github.com/HiFiiDev/BuildFenix/blob/master/changes.sh) for the *starting point* of the code, or I'll borrow from my changelog script.  There would be no self promotion

**Reply by HiFiiDev** [reply]
Make sure you read my response first I believe what I initially said was misinterpreted

**Reply by thelearn-tech** [reply]
Create that code with your  changelog script
And then pull request that . I will check and if ok I will merge pull request

**Comment by thelearn-tech** [comment]
Thanks man for following me it motivates a lot

**Reply by HiFiiDev** [reply]
You're welcome! Your responses are equally motivating me haha.  Download telegram so we can chat easier!!

**Reply by thelearn-tech** [reply]
Just add a pull request when u made the script

**Comment by HiFiiDev** [comment]
![Screenshot_20210116-230325](https://user-images.githubusercontent.com/46557883/104832561-21f07700-584f-11eb-84a7-10f9e6110d1f.png)
Hey there!! I'm working on it right now.
![Screenshot_20210116-230325](https://user-images.githubusercontent.com/46557883/104832561-21f07700-584f-11eb-84a7-10f9e6110d1f.png)

**Reply by thelearn-tech** [reply]
What is this Changelog thing??

**Reply by thelearn-tech** [reply]
I mean what it does
